src/app/dataScripts/fetchSpendingOverTimeData.ts

```diff
@@ -87,16 +104,17 @@ async function fetchSpendingData(
 }
 
 function cleanDataDump(
-    dataDump: MonthlySpendingOverTimeResponse
+    dataDump: MonthlySpendingOverTimeResponse,
+    population: number
 ): MonthlySpendingOverTime {
     // Aggregate spending value across months
     // Make String types into Number
-    const aggregatedOverMonths: MonthlySpendingOverTime = [];
+    const perCapitaSpendingOverMonths: MonthlySpendingOverTime = [];
     dataDump.results.reduce(
         (sum, { aggregated_amount, time_period: { fiscal_year, month } }) => {
             const aggregated = sum + aggregated_amount;
```

Comment:
```suggestion
            const aggregated = sum + ( aggregated_amount / population );
```
This is optional but moving the per capita calc for the month seems more clear to me up here since the purpose of the reduce fn is to aggregate the monthly spending with the previous month's, which is already a divided per capita number.

Reply:
I'll dig into the Wyoming bug. I think some undefineds might need to be coerced to 0.

Reply:
Oh, actually, it's just that the government's fiscal year starts in October. So April 2023 is actually month 6 of fiscal year 2023, not month 4.

Reply:
1723178